volpiano_display_utilities/latin_word_syllabification.py

```diff
@@ -241,7 +241,9 @@ def _get_syl_bound_position(ltrs_btw_vow_grps: str) -> Tuple[int, str]:
         else:
             split_case = "2 consonants between vowels (consonant group)"
     else:
-        if ltrs_btw_vow_grps[:2] in _CONSONANT_GROUPS:
+        if ltrs_btw_vow_grps[1:] in _CONSONANT_GROUPS:
+            syl_bound = num_ltrs_btw_vow_grps - 2
```

Comment:
could be worth adding a comment here with an example - `# in situations with 3 or more consonants in a row, group the final two consonants if possible (giving us "am-pli-us" rather than "amp-li-us", etc.)` or something similar.